module_utils/helpers.py

```
@@ -437,6 +437,24 @@ def get_control_console(socket_path):
     return Console(socket_path)
 
 
+def get_control_console_if_started(console_sock):
```

Seems that this function shouldn't return an error if it raises.
Or it should return an error only in case when it raises now.

---

I'll remove existing returned errors and change raise to return none, err